lib/fluent/plugin/out_s3_with_sqs.rb

```diff
@@ -190,8 +189,8 @@ def write(chunk)
 
       @after_flush.each_with_index do |after_flush_cmd, i|
         config_file = @after_flush_config[i]
-        after_hook_succeeded = system(after_flush_cmd, config_file, @s3_bucket, s3path, sqs_prefix)
```

Comment:
The only fluentd config that used the `sqs_prefix` field was experiment service, and @vijaykramesh has told me that we're not actually sending events to that fluentd tag anymore: https://github.com/change/deploy/blob/master/site-cookbooks/change-fluentd/templates/default/server/redshift_analytics.conf.erb#L65

I have an upcoming PR in the deploy repo to remove the fluentd matcher for `experiment_service.*` entirely since it's no longer being used and is confusing to have there